pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -7230,6 +7547,72 @@ def solve_model(model, options=None):
                     results_2 = solver2.solve(
                         model_h, options={"limits/gap": 0.3}, tee=True
                     )
+        elif model.config.hydraulics == Hydraulics.co_optimize_linearized:
+            # Currently, this method is supported for only MINLP solvers accessed through GAMS.
+            # The default solver is Baron and is set to find the first feasible solution.
+            # If the user has SCIP, then the feasible solution from BARON is passed to SCIP, which is then solved to a gap of 0.3.
+            # If the user do not have these solvers installed then it limits the use of co_optimize method at this time.
+            # Ongoing work is geared to address this limitation and will soon be updated here.
+
+            # Adding temporary variable bounds until the bounding method is implemented for the following Vars
+            model_h.v_F_Piped.setub(1050)
+            model_h.hydraulics.v_Pressure.setub(3.5e6)
+            # model_h.write("treatmentLinear.gms", io_options={'symbolic_solver_labels': True})
+
+            try:
+                # solver = SolverFactory("gams")
+                solver = SolverFactory("cbc")
```

Comment:
Why are you forcing CBC to be the solver? Can Gurobi be used if the user has it installed? If so, I think we should allow the user to use Gurobi if they wish.